tox.ini

```diff
@@ -27,16 +27,19 @@ commands =
 
 [testenv:docs]
 basepython = python3.9
+skip_install = true
```

Comment:
Your project has a little bit of a strange installation process. In your `setup.py` you define `requirements.txt` as `install_requires`. Both files server a different purpose, and usually should not be used like that.

With the above line you skip the installation. And as your requirements including `PySide2` are listed in the `requirements.txt`, it won't be installed.

When building the documentation, you import your application, see
https://github.com/Qiskit/qiskit-metal/blob/2f6a5f779e18f8940536743e479d39db08c22eab/docs/conf.py#L33

And your application tries to import `PySide2` - so it goes boom.

For me it looks like you need to delete the above line.

FWIW - I cannot build the documentation locally, even with the above line deleted.

If I may give you a tip: Python packaging has a steep learning curve, but now there is a really good book out there - which I wished would have existed 15 years ago:
https://www.manning.com/books/publishing-python-packages